### PR TITLE
docs: remove repetitive words in redis proxy docs

### PR DESCRIPTION
**Pull request by agrawalnishant**
Remove repetition of words ("to different") in Redis Proxy documentation.

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: docs/root/intro/arch_overview/other_protocols/redis.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ over consistency. This is the key point when comparing Envoy to `Redis Cluster
 <https://redis.io/topics/cluster-spec>`_. Envoy is designed as a best-effort cache,
 meaning that it will not try to reconcile inconsistent data or keep a globally consistent
 view of cluster membership. It also supports routing commands from different workload to
```
Comment (phlax) [inline docs/root/intro/arch_overview/other_protocols/redis.rst:11]: @agrawalnishant thanks for this - i think `workload` here should also be plural - ie `workloads` could you fix this also please

**Comment by phlax** [comment]
@agrawalnishant you will need to fix DCO too

**Comment by agrawalnishant** [comment]
Sorry @phlax merge history is probably not clean. Will create a new clean PR with these changes.

**Comment by agrawalnishant** [comment]
Sorry @phlax merge history is probably not clean. Will create a new clean PR with these changes.

**Pull request closed without merging**